src/stores/notifications/NotificationState.ts

```diff
@@ -83,10 +87,18 @@ export abstract class NotificationState
         }
     }
 
+    /**
+     * True if the notification has a count or a symbol and is greater than an activity notification.
+     */
     public get hasUnreadCount(): boolean {
         return this.level >= NotificationLevel.Notification && (!!this.count || !!this.symbol);
     }
 
+    /**
+     * True if the notification is a mention, an invitation, a knock or a unset message.
+     *
+     * @deprecated because the name is confusing. A mention is not an invitation, a knock or an unsent message.
```

Comment:
could you point to what someone using this should use instead?

Reply:
Fixed in [`2c46a04` (#29564)](https://github.com/element-hq/element-web/pull/29564/commits/2c46a04bd7bad74016ded8cedaad16e2dfe88f84)